### ISSUE TITLE
Verify coreclr libraries are the built ones

### ISSUE THREAD

**Issue by ThadHouse** [Tragetaschen/meta-aspnet#12]
I was using this setup to build for an armel platform, and got really close, but eventually ran into an issue that might be exposing itself in your build too. When I was doing the build, I noticed that the coreclr output binaries were not the ones built by the source-build, but instead were the ones corefx downloaded from NuGet directly. I noticed this because I was building soft float binaries, but the coreclr binaries in the image were hard float, which couldn't even be built by my toolchain. 

You should likely check with some hashes that the binaries included in your image are the built ones rather then the NuGet ones, otherwise, you might run into some odd issues.

**Comment by Tragetaschen** [comment]
Good point. I can see that for the existing ARM build as well, but I don't yet have an idea why…
